astropy/time/time_helper/function_helpers.py

```diff
@@ -0,0 +1,26 @@
+"""
+Helpers for overriding numpy functions in
+`~astropy.time.Time.__array_function__`.
+"""
+import numpy as np
+
+from astropy.units.quantity_helper.function_helpers import FunctionAssigner
+
+# TODO: Fill this in with functions that don't make sense for times
```

Comment:
Does this TODO need to be addressed before this can be merged? If not, do we need a follow-up issue?

Reply:
Good question – not quite sure what was intended to add there in #12703, but assuming it was to add some sort of graceful exit when calling a numpy function that cannot return sensible results for `Time`.
So I'd expect as long as this is unfilled, those functions will just continue to behave the same they are already doing now, so assuming it does not need to be completed right now.